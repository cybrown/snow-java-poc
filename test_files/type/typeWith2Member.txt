type IFoo {
    doSmth(value: int): int;
    doSmthElse(value: int, value2: string): long;
}
