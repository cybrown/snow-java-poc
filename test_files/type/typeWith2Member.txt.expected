<SnowType>
    <Identifier>IFoo</Identifier>
    <TypeInterface>
        <TypeInterfaceMember>
            <Identifier>doSmth</Identifier>
            <Argument>
                <Identifier>value</Identifier>
                <Identifier>int</Identifier>
            </Argument>
            <Identifier>int</Identifier>
        </TypeInterfaceMember>
        <TypeInterfaceMember>
            <Identifier>doSmthElse</Identifier>
            <Argument>
                <Identifier>value</Identifier>
                <Identifier>int</Identifier>
            </Argument>
            <Argument>
                <Identifier>value2</Identifier>
                <Identifier>string</Identifier>
            </Argument>
            <Identifier>long</Identifier>
        </TypeInterfaceMember>
    </TypeInterface>
</SnowType>
